functions-and-scope.js

```diff
@@ -50,15 +93,36 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomsten:
 // averageGrade(grades) geeft 6.642857142857143
-// averageGrade([6, 4, 5]) geeft xxxx
-// averageGrade([8, 9, 4, 6, 10]) geeft xxxx
+// averageGrade([6, 4, 5]) geeft 5
+// averageGrade([8, 9, 4, 6, 10]) geeft 7.4
 
+function averageGrade (gradeList) {
+    let totalGrade = null;
```

Comment:
Let dus wel op: met `null` geef je eigenlijk: er is geen enkele waarde, een `integer` zoals `0` is een waarde. 
Mogelijk kan dit ook met complexere functies errors gooien :)